openapi3/response.go

```diff
@@ -115,6 +115,9 @@ func (response *Response) Validate(ctx context.Context) error {
 	if response.Description == nil {
 		return errors.New("a short description of the response is required")
 	}
+	if xv := getValidationOptions(ctx).ExamplesValidation; !xv.Disabled {
```

Comment:
not sure if it's a good use of context switching flags back and forth while we traverse the spec